wdn/templates_5.0/js-src/navigation.babel.js

```diff
@@ -115,12 +112,13 @@ define(['plugins/headroom', 'plugins/body-scroll-lock'], function(Headroom, body
         if (window.matchMedia("(max-width: 56.12em)").matches) {
           skipNav.setAttribute('aria-hidden', 'false');
           institutionTitle.setAttribute('aria-hidden', 'false');
-          idm.setAttribute('aria-hidden', 'false');
-          search.setAttribute('aria-hidden', 'false');
           logo.setAttribute('aria-hidden', 'false');
           nav.setAttribute('aria-hidden', 'false');
```

Comment:
The navigation should be set to `aria-hidden="true"` when closed – _and_ by default when the window is `max-width: 56.12em`.